features/step_definitions/common.rb

```diff
@@ -61,8 +60,7 @@
 end
 
 When(/^I am completing a basic 18 or over "([^"]*)" conviction$/) do |value|
-  step %[I visit "/"]
-  step %[I click the "Start now" link]
+  step %[I have started an application]
```

Comment:
Super minor thing, but maybe instead of `an application` this might be a `a check`, for this service?